flink/v1.15/flink/src/main/java/org/apache/iceberg/flink/sink/IcebergFilesCommitter.java

```diff
@@ -233,6 +233,7 @@ public void notifyCheckpointComplete(long checkpointId) throws Exception {
     // the files,
     // Besides, we need to maintain the max-committed-checkpoint-id to be increasing.
     if (checkpointId > maxCommittedCheckpointId) {
+      LOG.info("Checkpoint {} completed. Attempting commit.", checkpointId);
```

Comment:
I remember that we merged this one: https://github.com/apache/iceberg/commit/5a4761c8fd227fa582b78d14f0a49de33b17b443

What happened with it?

Reply:
looks like a later backport unintentionally reverted it. I missed it during review. https://github.com/apache/iceberg/pull/6949